tmpl/volk_cpu.tmpl.c

```diff
@@ -73,7 +73,7 @@ struct VOLK_CPU volk_cpu;
 
 static inline unsigned int cpuid_count_x86_bit(unsigned int level, unsigned int count, unsigned int reg, unsigned int bit) {
 #if defined(VOLK_CPU_x86)
-    unsigned int regs[4] = {0};
+    int regs[4] = {0};
```

Comment:
Looks like the ```(int *)``` cast on the macro got lost, at least on my patch. So My MSVC specific macro looked like this ```#define cpuidex_x86(op, cnt, r) __cpuidex(((int*)r), op, cnt)```, then you dont need to change the regs type. But it ended up in the code like ```#define cpuid_x86_count(op, count, regs) __cpuidex(regs, op, count)```